src/libs/utils.js

```diff
@@ -343,3 +343,18 @@ export const sha256 = async message => {
     _.join('')
   )(new Uint8Array(hashBuffer))
 }
+
+export const formatBytes = bytes => {
+  if (bytes < 2 ** 10) {
+    return `${bytes} B`
+  }
+
+  const [prefix, divisor] = [
+    ['P', 2 ** 50],
+    ['T', 2 ** 40],
+    ['G', 2 ** 30],
+    ['M', 2 ** 20],
+    ['K', 2 ** 10]
+  ].find(([p, d]) => bytes >= d)
+  return `${(bytes / divisor).toPrecision(3)} ${prefix}iB`
+}
```

Comment:
I wonder if we should use an existing library (e.g. [filesize.js](https://filesizejs.com/)) instead of rolling our own.

Reply:
I don't know... If this was more complex, I'd agree. But is it worth vetting and maintaining a new dependency to replace 3 lines?

Reply:
I'm okay with not using an external library in this case.